File: digital_twin/day2/functions/nine.py
ga = 10

def funca(x, y):
    print("x =", x)
    print("y =", y)
    global ga
    ga = 22
    print("ga =", ga)

def funcb(a=10, b=20):
    print("a =", a)
    print("b =", b)
    print('ga =', ga)


#

File: slash/advC/day1/basics/nine.c
#include<stdio.h>
void funca(int la, int lb){
	printf("la=%d\n", la);
	printf("lb=%d\n", lb);
	return;
}


void main()
{
	int ma = 322;
	char ca = 'A';
	ca = (char)ma;
	printf("ca=%d\n", ca);
	printf("ca=%c\n", ca);
	ma = (int) ca;
	printf("ma=%d\n", ma);
	printf("ma=%c\n", ma);
	return;
}
